webcompat/static/js/lib/homepage.js

```diff
@@ -63,6 +64,39 @@ function HomePage() {
     });
   };
 
+  this.navbarHandler = function() {
+    var $navbar = $(".navigation");
+    var navbarHeight = $navbar.outerHeight();
+    var lastScrollY = window.pageYOffset;
+    var scrollTimeout = null;
+    var isScrolling = false;
+    $(window).on("scroll", function() {
+      isScrolling = true;
+      clearTimeout(scrollTimeout);
```

Comment:
I notice you're referring to the `setTimeout` and `setInterval` globals on the `window` object, but not `clearTimeout`. I don't have a hard preference either way (I normally just use global identifier w/o `window`), but it's probably good to be consistent.

Reply:
I'd probably be explicit and always to refer to `window`.